xla/service/gpu/ir_emitter_unnested.cc

```diff
@@ -2323,9 +2323,11 @@ absl::Status IrEmitterUnnested::EmitNcclThunk(
   }
 
   if (should_use_nccl_thunk) {
+    auto thunk_info = Thunk::ThunkInfo::WithProfileAnnotation(inst);
+    // when op is wrapped by syntactic sugar, use the wrapper name
```

Comment:
Nit: please make this a proper sentence (e.g. start with a capital letter and end with a .)